examples/2d/mesh2d_manual.rs

```diff
@@ -230,7 +230,6 @@ fn vertex(vertex: Vertex) -> VertexOutput {
     var out: VertexOutput;
     // Project the world position of the mesh into screen position
     out.clip_position = view.view_proj * mesh.model * vec4<f32>(vertex.position, 1.0);
-    // Unpack the `u32` from the vertex buffer into the `vec4<f32>` used by the fragment shader
```

Comment:
Why are you removing this?

Reply:
Sorry, This was not well synced with main. It should be resolved now I guess.